AssuredWorkloads/.repo-metadata.json

```diff
@@ -1,6 +1,6 @@
 {
     "distribution_name": "google/cloud-assured-workloads",
-    "release_level": "beta",
+    "release_level": "ga",
```

Comment:
We only have a v1beta1 client for assured workloads. Would you like for me to generate v1 first?

Reply:
I think for these it would be best to release V1 and wait a while, so I'll go ahead and revert them.